bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -752,7 +752,6 @@ impl Connection {
     /// client_hello.fingerprint_hash(FingerprintType::JA3, &mut hash);
     /// drop(conn);
     /// ```
-    #[cfg(feature = "unstable-fingerprint")]
     pub fn client_hello(&self) -> Result<&crate::client_hello::ClientHello, Error> {
```

Comment:
I'm all for not gating client_hello behind unstable-fingerprint, but don't we still need to gate the actual fingerprint methods? Aren't they now always available?